assets/schema_input.json

```diff
@@ -46,8 +46,14 @@
                 "meta": ["strandedness"],
                 "enum": ["unstranded", "forward", "reverse"],
                 "errorMessage": "Sample strandness valid input is unstranded,forward or reverse"
+            },
+            "sex": {
+                "type": "string",
+                "meta": ["sex"],
+                "enum": ["M", "F", "NA"],
+                "errorMessage": "The valid input for sample sex is M, F or NA"
             }
         },
-        "required": ["case", "sample", "fastq_1", "strandedness"]
+        "required": ["case", "sample", "fastq_1", "strandedness", "sex"]
```

Comment:
Could this become optional? And have a default of NA? That way it would be backwards compatible rather than break if some one used an old sample sheet

Reply:
Yes, that makes sense! I'll try adding it as a default.